source/MRMesh/miniply.cpp

```diff
@@ -1324,9 +1326,16 @@ namespace miniply {
         }
         ++m_pos;
       }
+      if ( commentLine )
+      {
+          while ( m_end < m_pos && is_whitespace( *m_end ) )
+              ++m_end; // skip white spaces after "comment" tag
+          m_comments.emplace_back( m_end, m_pos );
+      }
```

Comment:
mb use `MR::trim` instead?

Reply:
the style of whole file is to use raw pointers